insights/specs/datasources/yum_updates.py

```diff
@@ -213,6 +229,7 @@ def yum_updates(_broker):
               ]
             }
           },
+          "rebuild_pkgcache": false,
```

Comment:
I think this might be a typo, should it be `build_pkgcache`?

Other than this, everything looks good to me.

Reply:
thanks for noticing